modules/javafx.graphics/src/main/java/javafx/scene/image/WritableImage.java

```diff
@@ -75,6 +77,31 @@ public WritableImage(@NamedArg("width") int width, @NamedArg("height") int heigh
         super(width, height);
     }
 
+    private PixelBuffer pixelBuffer = null;
+
+    /**
+     * Construct a {@code WritableImage} using the provided {@code PixelBuffer}.
+     * The {@code java.nio.Buffer} provided by {@code PixelBuffer} will be used
+     * as pixel data for this image.
+     * {@code PixelBuffer} must be created with either {@code PixelFormat.INT_ARGB_PRE}
+     * or {@code PixelFormat.BYTE_BGRA_PRE} {@code PixelFormat}.
+     * The {@code PixelBuffer} can be shared by multiple {@code WritableImage}s
+     * and different applications.
```

Comment:
I don't know what you mean by "and different applications". You can remove that.